### DISCUSSION TITLE
it's slowly when open it

### DISCUSSION THREAD

**Discussion by ziyourenchina** [KRTirtho/spotube#1161] [Q&A]
Open the music home page, it shows very slowly,

**Comment by nepalivai** [comment]
Just force close the app and reopen that might fix the issue 
or
If this page is your problem that because you need to log in to you Spotify account. #1078 
<img width="638" alt="image" src="https://github.com/KRTirtho/spotube/assets/108126089/95b53533-b8c8-4cb3-b4b8-e8d8e204d8e9">
